.editorconfig

```diff
@@ -0,0 +1,14 @@
+# top-most EditorConfig file
+root = true
+
+# Unix-style newlines with a newline ending every file
+[*]
+end_of_line = lf
+insert_final_newline = true
+# Set default charset
+charset = utf-8
+
+# 2 space indentation
+[*.{ts,json}]
+indent_style = space
+indent_size = 2
```

Comment:
`trim_trailing_whitespace = true` ?

Reply:
you, right. Wanted to add, but forgot. Sure, no problem